app/views/test/components/iterations_summary_table/index.html.haml

```diff
@@ -0,0 +1 @@
+%div{data: {react: {"iterations-summary-table": true, data: {solution_id: @solution.id, iterations: @solution.serialized_iterations.to_json}}}}
```

Comment:
See note in the routes below. This will be refactored into a component helper next.